lib/asar/fs-wrapper.ts

```diff
@@ -382,10 +382,10 @@ export const wrapFsWithAsar = (fs: Record<string, any>) => {
 
   fs.promises.realpath = util.promisify(fs.realpath.native);
 
-  const { exists } = fs;
-  fs.exists = (pathArgument: string, callback: any) => {
+  const { exists: nativeExists } = fs;
+  fs.exists = function exists (pathArgument: string, callback: any) {
```

Comment:
Why not just `function` instead of `function exists`?

Reply:
@rzhao271 the precipitating PR is linked in the commit: https://github.com/nodejs/node/pull/43218